### PR TITLE
Lint website

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

The lint command is linting `js/_website` but the linted version is not in main yet so it modifies a lot of files unrelated to my PRs. I also added some directories to `.prettierignore` that should not be linted.

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests & Changelog

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

1. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`
  
1. Unless the pull request is labeled with the "no-changelog-update" label by a maintainer of the repo, all pull requests must update the changelog located in `CHANGELOG.md`:

Please add a brief summary of the change to the Upcoming Release section of the `CHANGELOG.md` file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

### CODE REVIEW

**Review by pngwn — APPROVED** [review]
Looks good!

We should add this to the ignore refs git thingy when it is merged. 

Thanks for handling this @freddyaboulton!

**Review by aliabd — APPROVED** [review]
Thanks @freddyaboulton, this looks good! :)

File: js/_website/src/routes/docs/docs.json
[the remark this answers is not in the document]
Comment (aliabd) [reply js/_website/src/routes/docs/docs.json:1]: Yes @pngwn is definitely right I need to remove this. I don't remember why I started tracking them in git. there was some reason... I'll clean this up soon.

**Comment by freddyaboulton** [comment]
Thanks @pngwn and @aliabd ! Will add this to the git ref log file tomorrow!

**Pull request merged**